solana/ts/idl/2_0_0/json/ntt_transceiver.json

```diff
@@ -2,6 +2,12 @@
   "version": "2.0.0",
   "name": "ntt_transceiver",
   "instructions": [
+    {
+      "name": "transceiverType",
```

Comment:
Is this available in ABI version 2.0.0 but just never made it in here?

Reply:
This was not in ABI version 2.0.0. I'll bump the ABI version and introduce add this in version 3.0.0